eidas_node/saml.py

```diff
@@ -518,7 +518,8 @@ def create_light_response(self) -> LightResponse:
         response.relay_state = self.relay_state
         return response
 
-    def _parse_assertion(self, response: LightResponse, assertion: Element) -> None:
+    def _parse_assertion(self, response: LightResponse, assertion: Element,
+                         auth_class_map: Optional[Dict[str, LevelOfAssurance]]) -> None:
```

Comment:
When you declare the type of argument and it's default value is `None`, it's redundant to mark it as `Optional`. `mypy` will derive this automatically.

Reply:
`auth_class_map` doesn't have a default value here.

Reply:
Ah, so it can be `None`, but it must be stated explicitly. OK.